deconz/Dockerfile

```diff
@@ -56,7 +56,8 @@ RUN if [ "${BUILD_ARCH}" = "armhf" ]; \
     && sed -i 's/\/root/\/data/' /etc/passwd
 
 COPY data/ika-otau-dl.sh /bin/
+COPY data/ledvance-otau-dl.sh /bin/
 COPY data/nginx.conf /etc/nginx/nginx.conf
 COPY data/run.sh data/discovery.sh /
 
-CMD ["/run.sh"]
+CMD ["/bin/chmod a+x /bin/ledvance-otau-dl.sh && /run.sh"]
```

Comment:
Setting permissions should be either in the Docker build or just ensure it is correctly set into the repository. Either way, not in the RUN command.

Reply:
Ok sorry, i’ll look to fix that this evening.

Reply:
Between that two methods, does there is one to be prefered ?

Reply:
I prefer to use git-index: `git update-index --chmod=+x path-name-shell-script`

Reply:
agreed.